microsetta_public_api/resources.py

```diff
@@ -22,6 +24,22 @@ def _dict_of_paths_to_alpha_data(dict_of_qza_paths, resource_name):
     return new_resource
 
 
+def _dict_of_paths_to_pcoa(dict_of_qza_paths, resource_name):
+    _validate_dict_of_paths(dict_of_qza_paths,
+                            resource_name)
+    new_resource = _replace_paths_with_qza(dict_of_qza_paths,
+                                           PCoAResults,
+                                           view_type=OrdinationResults)
+    return new_resource
+
+
+def _dict_of_dict_of_paths_to_pcoa(dict_of_dict_of_qza_paths, resource_name):
```

Comment:
A refactor for the future would be to create some objects to represent this complex data type

Reply:
Yes. For the milestone item "update results API to provide shotgun results in addition to amplicon" I intend to refactor this whole module so it is much more OOP